websites/B/Brain.fm/presence.ts

```diff
@@ -0,0 +1,111 @@
+const presence = new Presence({
+		clientId: "981442534167441428"
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", () => {
+	const presenceData: PresenceData = {
+		largeImageKey: "logo",
+		startTimestamp: browsingTimestamp
+	};
+
+	if (["www.brain.fm", "try.brain.fm"].includes(window.location.hostname)) {
+		if (["/", "/home"].includes(document.location.pathname))
+			presenceData.details = "Viewing the home page.";
```

Comment:
For the most part, we do not allow punctiation in the `state` or `details`. Update this throughout the presence
```suggestion
			presenceData.details = "Viewing the home page";
```